### PR TITLE
refactor: single transfer account page

### PR DESCRIPTION
**Pull request by tristanhcole**
**Describe the Pull Request**
Refactor single transfer account page to ant design. This PR also swaps credit transfer list on the single transfer account page to use new serverside search and filtering. This means accounts with a large number of payments 10K + will now load quickly.

Screen Shot 2021-07-20 at 5 40 14 pm

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver
- [ ] Add release notes to [unreleased changelog](https://github.com/teamsempo/SempoBlockchain/blob/master/CHANGELOG.md#unreleased)

### CODE REVIEW

**Comment by tristanhcole** [comment]
[Refactor single transfer account page](https://trello.com/c/mDvmFwLE/862-refactor-single-transfer-account-page)

**Review by michielderoos — APPROVED** [review]
Looks great!  
As a nice/potentially useful side effect of this PR, `/accounts/1` (the org-level account) now loads quickly and behaves responsively as well

File: app/server/schemas.py [hunk 1]
```diff
@@ -493,10 +494,8 @@ class DisbursementSchema(SchemaBase):
         "credit_sends.recipient_transfer_account",
         "credit_receives.sender_transfer_account",
         "credit_receives.recipient_transfer_account",
-        # "credit_sends.sender_user",
```
[the remark this answers is not in the document]
Comment (tristanhcole) [reply app/server/schemas.py:496]: it does delete comments

**Comment by tristanhcole** [comment]
> Looks great!
> As a nice/potentially useful side effect of this PR, `/accounts/1` (the org-level account) now loads quickly and behaves responsively as well

Good point!

**Pull request merged**